### PR TITLE
chat: send more facts to / subscriptions

### PR DESCRIPTION
**Pull request by Fang-**
Chat was already sending some updates (block & msg toggle changes) to the top-level subscription path (and surprisingly isn't sending those anywhere else), but wasn't doing the same for updates about individual chats.

Here, we chuck `/` into the list of paths to update for most response-level subscription updates, to mirror the "firehose" style that channels' `/` endpoint has.

Closes LAND-1880.

### CODE REVIEW

**Comment by arthyn** [comment]
I think you want to add them for `give-unread` and `di-give-writs-diff` as well

**Review by latter-bolden — APPROVED** [review]
Looking good! I'll work on listening for live updates & moving existing subs over once the DM create/invite  PR lands.

re unreads: having them in the firehose makes sense to me, but isn't critical for our needs on the client. Will defer to y'all on that one

**Pull request merged**
